ui/angular.json

```diff
@@ -24,13 +24,9 @@
             ],
             "styles": [
               "src/styles.css",
-              "node_modules/bootstrap/dist/css/bootstrap.min.css",
```

Comment:
Don't we have to use clarity styles? Just use the module as you import it into the app.module.ts?

Reply:
Yes i found that this isn't necessary when using clarity-angular